api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeHttpConfig.java

```diff
@@ -49,8 +55,18 @@ public KafkaBridgeHttpConfig(int port) {
         this.port = port;
     }
 
+    @Description("The host which is the server listening on.")
+    @JsonProperty(defaultValue = HTTP_DEFAULT_HOST)
+    public String getHost() {
+        return host;
+    }
+
+    public void setHost(String host) {
+        this.host = host;
+    }
```

Comment:
What would be the point of the host configuration? We are here only in Kubernetes context, so I'm not sure what would you use it for.

Reply:
Reason is that @ppatierno was mentioning that `host` is currently supported, but I guess it was not as it was not accessible though Kubernetes API: https://github.com/strimzi/strimzi-kafka-operator/issues/8732#issue-1774631381 also wrote some comments and concerns in the issue.

Reply:
I would agree. That was the reason it was taken out.

Reply:
Maybe I was writing my comment while Steffen was writing his comment :-)
I meant to agree with Jakub, so we should taken it out.

Reply:
Will revert those changes, no problem :)